arbitrum-docs/stylus/stylus-quickstart.md

```diff
@@ -130,65 +217,81 @@ Caused by:
+
+```bash
 cargo stylus deploy \
   --private-key-path=<PRIVKEY_FILE_PATH> \
   --estimate-gas-only
 ```
 
-and see:
+Among other details, you'll see an estimated gas cost for deployment:
 
-```
-Compressed WASM size: 3 KB
-Deploying program to address 0x457b1ba688e9854bdbed2f473f7510c476a3da09
-Estimated gas: 12756792
+```bash
+...
+====DEPLOYMENT====
+Deploying program to address 0xD6bC450ED8D8054b7c971f6d42631Fc4ed59A099
+Base fee: 0.100000000 gwei
+Estimated gas for deployment: 11266639 gas units
 ```
 
-Next, attempt an actual deployment. Two transactions will be sent onchain.
+To proceed with the deployment, two transactions will be sent onchain:
 
-```
+```bash
 cargo stylus deploy \
   --private-key-path=<PRIVKEY_FILE_PATH>
 ```
 
-and see:
+You'll see details about the deployment process, such as:
 
+```bash
+...
+Uncompressed WASM size: 25.0 KB
+Compressed WASM size to be deployed onchain: 9.0 KB
+Connecting to Stylus RPC endpoint: https://stylus-testnet.arbitrum.io/rpc
+Program succeeded Stylus onchain activation checks with Stylus version: 1
+Deployer address: 0x44E623584f90D6FBaBe930221d47E0De4248e935
+
+====DEPLOYMENT====
+Deploying program to address 0xD6bC450ED8D8054b7c971f6d42631Fc4ed59A099
+Base fee: 0.100000000 gwei
+Estimated gas for deployment: 11266639 gas units
+Submitting deployment tx...
+Confirmed deployment tx 0xc95d7fab996934b38d2eb5e4559977aa222a658fb13e994bdea0f74746219ad5
+Gas units used 10318928, effective gas price 0.100000000 gwei
+Transaction fee: 0.001031892800000000 ETH
+
+====ACTIVATION====
+Activating program at address 0xD6bC450ED8D8054b7c971f6d42631Fc4ed59A099
+Base fee: 0.100000000 gwei
+Estimated gas for activation: 14224495 gas units
+Submitting activation tx...
+Confirmed activation tx 0x908459dab815e1403c36e21838673ac55877665c9d95c9c87bfb1c4543eab0e8
+Gas units used 14184722, effective gas price 0.100000000 gwei
+Transaction fee: 0.001418472200000000 ETH
 ```
-Compressed WASM size: 3 KB
-Deploying program to address 0x457b1ba688e9854bdbed2f473f7510c476a3da09
-Estimated gas: 12756792
-Submitting tx...
-Confirmed tx 0x42db…7311, gas used 11657164
-Activating program at address 0x457b1ba688e9854bdbed2f473f7510c476a3da09
-Estimated gas: 14251759
-Submitting tx...
-Confirmed tx 0x0bdb…3307, gas used 14204908
 
-```
+You can verify the deployment and activation of your program on the [Stylus testnet block explorer](https://stylus-testnet-explorer.arbitrum.io/address/0xD6bC450ED8D8054b7c971f6d42631Fc4ed59A099/transactions#address-tabs).
+
+For further deployment options and transaction details, consult `cargo stylus deploy --help`.
 
-More options exist for sending and outputting your transaction data. See `cargo stylus deploy --help` for more details.
 
-## Deploying non-Rust WASM projects
+## Deploying Non-Rust WASM Projects
```

Comment:
Other than this line I'd remove the changes in this section. It was meant as a short aside, and it probably should be moved to another part of the docs altogether since it's focused on WASM and this page is meant as a Rust quick start guide. We'll revisit closer to mainnet.